api/auth.py

```diff
@@ -20,12 +20,22 @@ class TokenData(BaseModel):
     username: Optional[str] = None
 
 
+class Settings(BaseSettings):
+    secret_key: str
+
+    class Config:
+        env_file = "../.env"
+
+
 class Authentication:
     # ToDo: use settings https://fastapi.tiangolo.com/advanced/settings/
 
+    settings = Settings()
+    print("secret_key :", settings.secret_key)
```

Comment:
This may be useful to you to check the changes are working, but please remove it before the PR can be merged.